packages/protocol/contracts/ProviderStore.sol

```diff
@@ -154,27 +141,20 @@ contract ProviderStore is RequesterStore, IProviderStore {
     /// @param providerId Provider ID
     /// @return admin Provider admin
     /// @return xpub Master public key of the provider node
+    /// @return authorizers Authorizer contract addresses
     function getProvider(bytes32 providerId)
```

Comment:
I see this doesn't have unit tests. Worth testing too?

Reply:
The unit tests are not complete in general, but okay leaving this as a note